adobe-esign-connector-demo/src_hd/com/axonivy/connector/adobe/esign/connector/demo/Demo/Demo.xhtml

```diff
@@ -1,88 +1,193 @@
-      </h3>
+		<ui:define name="title">Demo</ui:define>
+		<ui:define name="content">
 
-      <h:form id="form" enctype="multipart/form-data">
-      	<p:remoteCommand name="updateDocument" actionListener="#{logic.updateDocuments}" update="@form" />
-        <p:messages />
-        <p:panelGrid columns="2" layout="grid" styleClass="ui-panelgrid-blank ui-fluid" 
-          columnClasses="ui-g-12 ui-md-3 ui-lg-2, ui-g-12 ui-md-9 ui-lg-4, ui-g-12 ui-md-3 ui-lg-2, ui-g-12 ui-md-9 ui-lg-4">
+			<script type="text/javascript">
+				function closeSigningDlg() {
+					PF('dlgIframe').hide();
+					updateDocument();
+				}
+			</script>
+			<h3>
+				#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/demo/Demo/AdobeSignDemosA')}
+			</h3>
 
-			<h:outputText value="Select a file to upload and sign" />
-            <p:fileUpload value="#{data.file}" mode="simple" id="fileUpload" skinSimple="true" />
-            
-			<p:outputLabel for="@next" value="Signer 1" />
-			<p:inputText value="#{data.signer1}" />
-			
-			<p:outputLabel for="@next" value="Signer 2" />
-			<p:inputText value="#{data.signer2}" />
-			
-			<p:outputLabel for="@next" value="Agreement ID" />
-        	<p:inputText id="agreementId" value="#{data.agreementId}" />
+			<h:form id="form" enctype="multipart/form-data">
+				<p:remoteCommand name="updateDocument"
+					actionListener="#{logic.updateDocuments}" update="@form" />
+				<p:messages />
+				<p:fieldset
+					legend="#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/demo/Demo/PDFFile')}">
+					<div class="ui-fluid ">
+						<div class="p-field p-grid p-align-baseline">
+							<div class="p-col-12 p-md-6">
+								<p:outputLabel
+									value="#{ivy.cms.co('/Dialogs/com/axonivy/connector/adobe/esign/connector/demo/Demo/PDFFileToSign')}" />
+							</div>
+							<div class="p-col-12 p-md-6">
+								<p:fileUpload value="#{data.file}" mode="simple" id="fileUpload"
+									skinSimple="true" >
+								</p:fileUpload>
+							</div>
+						</div>
+					</div>
+				</p:fieldset>
 
-			<p:commandButton value="Create Agreement 1" actionListener="#{logic.demo1Signer}" update="@form" process="@form" />
-        	<p:outputLabel value="creates a new agreement for 1 signer " />
-        	
-        	<p:commandButton value="Create Agreement 2" actionListener="#{logic.demo2Signer}" update="@form" process="@form" />
-        	<p:outputLabel value="creates a new agreement for 2 signers " />
-        	
-        	<p:commandButton value="Load existing Agreement" actionListener="#{logic.updateDocuments}" update="@form" process="@form" />
-        	<p:outputLabel value="loads the data for an existing agreement" />
+				<!-- 			<p:fieldset legend="PDF file"> -->
+				<!-- 				<h:outputText value="Select a file to upload and sign" /> -->
+				<!-- 	            <p:fileUpload value="#{data.file}" mode="simple" id="fileUpload" skinSimple="true" /> -->
+				<!-- 			</p:fieldset> -->
```

Comment:
Hi @stefanmasek 
Can we remove this commented block code completely or there was any concern/ issue that we need more time to investigate?

Reply:
sure, will remove it